From: Benjamin Drung <bdrung@debian.org>
Date: Thu, 28 Dec 2017 16:23:49 +0000
Subject: Fix bad whatis entries of manpages

Each manual page should start with a "NAME" section, which lists the
name and a brief description of the page separated by "\-". The "NAME"
section is parsed by lexgrog and used to generate a database that's
queried by commands like apropos and whatis. This tag indicates that
lexgrog was unable to parse the NAME section of this manual page.
.
For manual pages that document multiple programs, functions, files, or
other things, the part before "\-" should list each separated by a
comma and a space. Each thing listed must not contain spaces; a man
page for a two-part command like "fs listacl" must use something like
"fs_listacl" in the "NAME" section so that it can be parsed by
lexgrog.
.
Refer to the lexgrog(1) manual page, the groff_man(7) manual page, and
the groff_mdoc(7) manual page for details.
---
 man/man3/fi_getinfo.3  | 4 ++--
 man/man7/fabric.7      | 2 +-
 man/man7/fi_bgq.7      | 2 +-
 man/man7/fi_direct.7   | 2 +-
 man/man7/fi_gni.7      | 2 +-
 man/man7/fi_mlx.7      | 2 +-
 man/man7/fi_provider.7 | 2 +-
 man/man7/fi_psm.7      | 2 +-
 man/man7/fi_psm2.7     | 2 +-
 man/man7/fi_sockets.7  | 2 +-
 man/man7/fi_udp.7      | 2 +-
 man/man7/fi_usnic.7    | 2 +-
 man/man7/fi_verbs.7    | 2 +-
 13 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/man/man3/fi_getinfo.3 b/man/man3/fi_getinfo.3
index 9e63df7..180b501 100644
--- a/man/man3/fi_getinfo.3
+++ b/man/man3/fi_getinfo.3
@@ -1,9 +1,9 @@
 .TH "fi_getinfo" "3" "2017\-07\-25" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-fi_getinfo / fi_freeinfo \- Obtain / free fabric interface information
+fi_getinfo, fi_freeinfo \- Obtain / free fabric interface information
 .PP
-fi_allocinfo / fi_dupinfo \- Allocate / duplicate an fi_info structure
+fi_allocinfo, fi_dupinfo \- Allocate / duplicate an fi_info structure
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/man/man7/fabric.7 b/man/man7/fabric.7
index 16fb0bd..1e204ff 100644
--- a/man/man7/fabric.7
+++ b/man/man7/fabric.7
@@ -1,7 +1,7 @@
 .TH "fabric" "7" "2016\-06\-29" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-Fabric Interface Library
+fabric \- Fabric Interface Library
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/man/man7/fi_bgq.7 b/man/man7/fi_bgq.7
index e8b6276..915fc4d 100644
--- a/man/man7/fi_bgq.7
+++ b/man/man7/fi_bgq.7
@@ -1,7 +1,7 @@
 .TH "fi_bgq" "7" "2017\-02\-27" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The Blue Gene/Q Fabric Provider
+fi_bgq \- The Blue Gene/Q Fabric Provider
 .SH OVERVIEW
 .PP
 The bgq provider is a native implementation of the libfabric interfaces
diff --git a/man/man7/fi_direct.7 b/man/man7/fi_direct.7
index f9039fd..d12bdd4 100644
--- a/man/man7/fi_direct.7
+++ b/man/man7/fi_direct.7
@@ -1,7 +1,7 @@
 .TH "fi_direct" "7" "2017\-04\-13" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-Direct fabric provider access
+fi_direct \- Direct fabric provider access
 .SH SYNOPSIS
 .IP
 .nf
diff --git a/man/man7/fi_gni.7 b/man/man7/fi_gni.7
index b831cd6..3553d97 100644
--- a/man/man7/fi_gni.7
+++ b/man/man7/fi_gni.7
@@ -1,7 +1,7 @@
 .TH "fi_gni" "7" "2017\-07\-21" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The GNI Fabric Provider
+fi_gni \- The GNI Fabric Provider
 .SH OVERVIEW
 .PP
 The GNI provider runs on Cray XC (TM) systems utilizing the user\-space
diff --git a/man/man7/fi_mlx.7 b/man/man7/fi_mlx.7
index 6b81e89..dafce77 100644
--- a/man/man7/fi_mlx.7
+++ b/man/man7/fi_mlx.7
@@ -1,7 +1,7 @@
 .TH "fi_mlx" "7" "2017\-04\-18" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The MLX Fabric Provider
+fi_mlx \- The MLX Fabric Provider
 .SH OVERVIEW
 .PP
 The \f[I]mlx\f[] provider runs over the UCX library that is currently
diff --git a/man/man7/fi_provider.7 b/man/man7/fi_provider.7
index caa5220..1b86f7d 100644
--- a/man/man7/fi_provider.7
+++ b/man/man7/fi_provider.7
@@ -1,7 +1,7 @@
 .TH "fi_provider" "7" "2017\-07\-24" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-Fabric Interface Providers
+fi_provider \- Fabric Interface Providers
 .SH OVERVIEW
 .PP
 Conceptually, a fabric provider may be viewed as a local hardware NIC
diff --git a/man/man7/fi_psm.7 b/man/man7/fi_psm.7
index 21f14d6..86bbd5c 100644
--- a/man/man7/fi_psm.7
+++ b/man/man7/fi_psm.7
@@ -1,7 +1,7 @@
 .TH "fi_psm" "7" "2016\-02\-28" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The PSM Fabric Provider
+fi_psm \- The PSM Fabric Provider
 .SH OVERVIEW
 .PP
 The \f[I]psm\f[] provider runs over the PSM 1.x interface that is
diff --git a/man/man7/fi_psm2.7 b/man/man7/fi_psm2.7
index 2c67e1a..89b1c5e 100644
--- a/man/man7/fi_psm2.7
+++ b/man/man7/fi_psm2.7
@@ -1,7 +1,7 @@
 .TH "fi_psm2" "7" "2017\-07\-05" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The PSM2 Fabric Provider
+fi_psm2 \- The PSM2 Fabric Provider
 .SH OVERVIEW
 .PP
 The \f[I]psm2\f[] provider runs over the PSM 2.x interface that is
diff --git a/man/man7/fi_sockets.7 b/man/man7/fi_sockets.7
index 3b48ecc..7587882 100644
--- a/man/man7/fi_sockets.7
+++ b/man/man7/fi_sockets.7
@@ -1,7 +1,7 @@
 .TH "fi_sockets" "7" "2017\-08\-09" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The Sockets Fabric Provider
+fi_sockets \- The Sockets Fabric Provider
 .SH OVERVIEW
 .PP
 The sockets provider is a general purpose provider that can be used on
diff --git a/man/man7/fi_udp.7 b/man/man7/fi_udp.7
index fad5495..13037b9 100644
--- a/man/man7/fi_udp.7
+++ b/man/man7/fi_udp.7
@@ -1,7 +1,7 @@
 .TH "fi_udp" "7" "2017\-07\-27" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The UDP Fabric Provider
+fi_udp \- The UDP Fabric Provider
 .SH OVERVIEW
 .PP
 The UDP provider is a basic provider that can be used on any system that
diff --git a/man/man7/fi_usnic.7 b/man/man7/fi_usnic.7
index ae584cf..53d5bb0 100644
--- a/man/man7/fi_usnic.7
+++ b/man/man7/fi_usnic.7
@@ -1,7 +1,7 @@
 .TH "fi_usnic" "7" "2017\-07\-24" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The usNIC Fabric Provider
+fi_usnic \- The usNIC Fabric Provider
 .SH OVERVIEW
 .PP
 The \f[I]usnic\f[] provider is designed to run over the Cisco VIC
diff --git a/man/man7/fi_verbs.7 b/man/man7/fi_verbs.7
index b0608a4..4f87e7a 100644
--- a/man/man7/fi_verbs.7
+++ b/man/man7/fi_verbs.7
@@ -1,7 +1,7 @@
 .TH "fi_verbs" "7" "2017\-07\-24" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
 .SH NAME
 .PP
-The Verbs Fabric Provider
+fi_verbs \- The Verbs Fabric Provider
 .SH OVERVIEW
 .PP
 The verbs provider enables applications using OFI to be run over any
